docs/src/main/sphinx/develop/connectors.rst

```diff
@@ -79,33 +79,102 @@ features, like:
 Note that data modification also requires implementing
 a :ref:`connector-page-sink-provider`.
 
+When a ``SELECT`` query is being executed, and it contains specific clauses,
+the engine can try pushing down some of them into the connector, by calling
+one of the following methods of the ``ConnectorMetadata`` service:
+
+* ``applyLimit`` for the ``LIMIT`` clause, or ``applyTopN`` when ``ORDER BY``
+  is also present
+* ``applyFilter`` for the ``WHERE`` clause
+* ``applyProjection`` for expressions in the ``SELECT`` clause
+* ``applySample`` for the ``TABLESAMPLE`` clause
+* ``applyAggregation`` for the ``GROUP BY`` clause
+* ``applyJoin`` for the different types of the ``JOIN`` clause
+* ``applyTableFunction`` for the ``TABLE ()`` clause
```

Comment:
IIRC `TABLE ( )` is not PTF. It's just syntax sugar to wrap something in `SELECT * FROM`. e.g. try `TABLE orders` in trino-cli and it'll execute `SELECT * FROM orders`.

Or maybe I'm confused.

Reply:
I thought adding parenthesis would be enough, but I see how that can be confusing. BTW `TABLE <name>` doesn't work for subqueries:
```
trino:v2> table (select 1);
Query 20220725_063410_00002_jik9j failed: line 1:7: mismatched input '('. Expecting: <identifier>
table (select 1)
```

How about something like this?
```suggestion
* ``applyTableFunction`` for the ``TABLE ()`` table function clause
```

Reply:
Thanks for explaining. Your suggestion definitely seems better.